legacy/src/main/java/org/opensearch/sql/legacy/rewriter/nestedfield/From.java

```diff
@@ -70,29 +70,31 @@ private void keepParentTableOnly() {
     left().expr.setParent(query);
   }
 
-    /**
-     * <pre>
-     * Collect path alias and full path mapping of nested field in FROM clause.
-     * Sample:
-     * FROM team t, t.employees e ...
-     * <p>
-     * Join
-     * /    \
-     * team t    Join
-     * /    \
-     * t.employees e  ...
-     * <p>
-     * t.employees is nested because path "t" == parentAlias "t"
-     * Save path alias to full path name mapping {"e": "employees"} to Scope
-     * </pre>
-     */
-    private void collectNestedFields(Scope scope) {
-        From clause = this;
-        for (; clause.isCommaJoin(); clause = clause.right()) {
-            clause.left().addIfNestedField(scope);
-        }
-        clause.addIfNestedField(scope);
+  /**
+   *
+   *
```

Comment:
Looks like we have the same issue with 2 asterisks that Andrew pointed out on another spotless PR, did we figure out what caused this and how it was fixed?